source/UF/ConnectedTypes.lagda

```diff
@@ -0,0 +1,223 @@
+Ian Ray, 23rd July 2024
+
+We will define connected types and maps (recall our convetion for H-levels starts
```

Comment:
* _convention_ (Applies elsewhere too. Maybe enable a spell-checker to catch these and other misspellings.)
* _that H-levels_
* _We define_ sounds more natural to me and is in line with _we explore_.